linera-views/src/common.rs

```diff
@@ -108,14 +108,14 @@ pub fn get_interval(key_prefix: Vec<u8>) -> (Bound<Vec<u8>>, Bound<Vec<u8>>) {
 }
 
 pub(crate) fn from_bytes_opt<V: DeserializeOwned, E>(
-    key_opt: Option<Vec<u8>>,
```

Comment:
Technically that was a "move" (i.e. shallow copy).

Reply:
Ok.